examples/conftest/README.md

```diff
@@ -0,0 +1,83 @@
+# Conftest Example
+
+This example shows how to set cost policies using [Conftest](https://www.conftest.dev/).  For simplicity, this is based off the terraform-plan-json example, which does not require Terraform to be installed.
+
+Create a policy file that checks the Infracost JSON: 
+```rego
+package main
+
+deny_totalDiff[msg] {
+	maxDiff = 1500.0
+	to_number(input.diffTotalMonthlyCost) >= maxDiff
+
+	msg := sprintf(
+		"Total monthly cost diff must be < $%.2f (actual diff is $%.2f)",
+		[maxDiff, to_number(input.diffTotalMonthlyCost)],
+	)
+}
+
+deny_instanceCost[msg] {
+	r := input.projects[_].breakdown.resources[_]
+	startswith(r.name, "aws_instance.")
+
+	maxHourlyCost := 2.0
+	to_number(r.hourlyCost) > maxHourlyCost
+
+	msg := sprintf(
+		"AWS instances must cost less than $%.2f\\hr (%s costs $%.2f\\hr).",
+		[maxHourlyCost, r.name, to_number(r.hourlyCost)],
+	)
+}
+
+deny_instanceCost[msg] {
+	r := input.projects[_].breakdown.resources[_]
+	startswith(r.name, "aws_instance.")
+
+	baseHourlyCost := to_number(r.costComponents[_].hourlyCost)
+
+	sr_cc := r.subresources[_].costComponents[_]
+	sr_cc.name == "Provisioned IOPS"
```

Comment:
This is cool! It made me think back to cost component names and how the string we put in brackets, e.g. `(on-demand, db.t3.medium)` could be structured as key=value pairs in our JSON (but still output comma-separated in the `table` format) so policy conditions can use them too. Would people need cost policies against those attributes? 🤷

Reply:
yeah that's a good idea, I'll make a note in the 'update json ticket'.  I'm looking at OPA now and one thing it does is let you provide a json schema I guess so you can validate that your rego queries are legit.